Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1354,7 +1354,7 @@ class RunWebDriverTests(shell.TestNewStyle, CustomFlagsMixin, ShellMixin):
     description = ["webdriver-tests running"]
     descriptionDone = ["webdriver-tests"]
     jsonFileName = "webdriver_tests.json"
-    command = ["python3", "Tools/Scripts/run-webdriver-tests", "--json-output={0}".format(jsonFileName), WithProperties("--%(configuration)s")]
+    command = ["python3", "Tools/Scripts/run-webdriver-tests", "--verbose", "--json-output={0}".format(jsonFileName), WithProperties("--%(configuration)s")]
```

Comment:
can you also replace .format() with f-string while you are at all.

Reply:
Good catch. I'll do it.